manifests/initrd.pp

```diff
@@ -0,0 +1,21 @@
+#
+# == Class: kmod::initrd
+#
+# Reconfigures the initramfs image after modifying kernel module configuration.
+#
+#
+class kmod::initrd(Optional[String] $cmd = $facts.kmod_update_initrd_cmd){
+  if defined($cmd) {
+    exec { 'Update initramfs image.':
```

Comment:
May want to make this `"${module_name}: update initramfs"` to be sure that it can't conflict with anything else.

Reply:
gotcha.

Reply:
Done.